librz/include/rz_analysis.h

```diff
@@ -1468,12 +1468,12 @@ typedef struct {
 	ut64 value; ///< data either written to or read from
 } RzILTraceRegOp;
 
-enum {
+typedef enum {
 	RZ_IL_TRACE_INS_HAS_MEM_R = 0x1U, ///< instruction include memory read
```

Comment:
```suggestion
	RZ_IL_TRACE_INS_OP_MEM_R = 0x1U, ///< instruction include memory read
```
?

Reply:
I did think of removing the `HAS` part but then I saw the original usage of the identifier in [librz/analysis/il_trace.c](https://github.com/rizinorg/rizin/blob/1354473f81c645e72414a73d4322ab5c9ec704d5/librz/analysis/il_trace.c) and I thought that the `HAS` is pretty appropriate for the algorithms being used there ... and I'm just writing a static method so I might as well not rename it.

Reply:
Yes, lets keep it as is for now.